web-app/prisma/schema.prisma

```diff
@@ -353,23 +354,98 @@ enum AgentListType {
   FAVORITE
 }
 
+model CreditTransaction {
+  id            String                @id @default(cuid())
+  createdAt     DateTime              @default(now())
+  updatedAt     DateTime              @updatedAt
+  amount        BigInt //positive is always a credit topup and negative is always a credit spend
+  includedFee   BigInt
+  note          String?
+  noteKey       String?
+  errorNote     String?
+  errorNoteKey  String?
```

Comment:
What is a `note` and what is a `noteKey`? Same for error. I don't know the context.

Reply:
note is the error message(in english), noteKey is some key for the specific error which can be used in the translation lib (please see the error descriptions I added)